src/sentry/static/sentry/app/views/settings/components/forms/formField/index.tsx

```diff
@@ -182,6 +213,8 @@ class FormField extends React.Component {
     const model = this.getModel();
 
     if (onChange) {
+      //TODO(TS): fix properly
+      // @ts-ignore
       onChange(value, event);
```

Comment:
You can often use the `callIfFunction` util function to get around these type errors and conditional blocks.

Reply:
@markstory Not sure why I had this here originally, turns out I don't need the `@ts-ignore`.